.github/CODEOWNERS

```diff
@@ -1,2 +1,38 @@
 # Review from a member of the review-team is required
 * @mskcc-omics-workflows/reviewers
+./modules/msk/custom/splitfastqbylane/** @anoronh4 @mskcc-omics-workflows/reviewers
+./modules/msk/facets/** @nikhil @mskcc-omics-workflows/reviewers
+./modules/msk/gatk4/applybqsr/** @yocra3 @FriederikeHanssen @mskcc-omics-workflows/reviewers
```

Comment:
Just fyi we need to fix the meta yaml files here  by removing these users: @yocra3 @FriederikeHans